resolver-dns/src/main/java/io/netty/resolver/dns/DefaultDnsCache.java

```diff
@@ -168,38 +213,13 @@ public DnsCacheEntry cache(String hostname, DnsRecord[] additionals, Throwable c
     private void cache0(String hostname, DefaultDnsCacheEntry e, int ttl, EventLoop loop) {
```

Comment:
while you are here :) `MAX_SUPPORTED_TTL_SECS` is used on line 194 and 209 ... but it looks unnecessary because it is already enforced in the constructor. consider removing.